src/main/java/com/alyrow/gdx/particle/physics/ImageColour.java

```diff
@@ -0,0 +1,53 @@
+package com.alyrow.gdx.particle.physics;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.math.MathUtils;
+import com.badlogic.gdx.math.Vector2;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+public class ImageColour extends PhysicForce {
+
+    private Color[][] matrix;
+
+    public ImageColour(BufferedImage image) {
+        matrix = new Color[image.getWidth()][image.getHeight()];
+        java.awt.Color c;
+        for (int i = 0; i < image.getWidth(); i++) {
+            for (int j = 0; j < image.getHeight(); j++) {
+                c = new java.awt.Color(image.getRGB(i, j));
+                matrix[i][matrix[0].length - j - 1] = new Color(c.getRed() / 256f, c.getGreen() / 256f, c.getBlue() / 256f, 1f);
+            }
+        }
+    }
```

Comment:
`java.awt.Color c` can simply be replaced with libGDX color
I could suggest to replace `BufferedImage` by a `Pixmap` with the image loaded inside

```suggestion
    public ImageColour(Pixmap image) {
        matrix = new Color[image.getWidth()][image.getHeight()];
        Color c = new Color();
        for (int i = 0; i < image.getWidth(); i++) {
            for (int j = 0; j < image.getHeight(); j++) {
                c.set(image.getPixel(x, y));
                matrix[i][matrix[0].length - j - 1] = new Color(c.r, c.g, c.b, 1f);
            }
        }
    }
```